doc/adrs/124_support_deck_representation.adoc

```diff
@@ -0,0 +1,168 @@
+* *Delete*: If configured in the view model, it is possible to delete a Lane.
+
+
+
+=== Actions associated to Cards
+
+* *Selection*: When clicking on the `Card` the properties of the associated semantic element are dispayed in `Details` and the semantic element is selected in the `Explorer`.
+* *Drag&Drop*: if a Drag&Drop tool has been defined in the View model, it is possible to
+** drag a Card and drop it beside another Card in the same Lane.
+** drag a Card and drop it beside another Card in another Lane.
+* *Edit*: If configured in the view model, a dialog is opened and allows to change the Title, Label and Description of the Card.
+* *Hide*: If configured in the view model, it is possible to hide a Card. This information is stored in the representation.
+* *Delete*: If configured in the view model, it is possible to delete a Card.
+
+
+
+=== Concepts of Deck description
+
+`DeckDescription` is a new concept in the *View model* to describe the Deck representation.
+`DeckDescription` is associated to a object type in a chosen meta-model and a precondition expression to control on which object in the model the Gantt can be created.
+
+It contains `LaneDescription` which allow to define the lane
+
+A lane is associated to a semantic element : in this case the element is defined thanks to a semantic expression on the `LaneDescription`. The `LaneDescription` will produce as many Lane as the number of semantic objects returned by the semantic candidates expression. The Lane will be displayed in the order of the returned semantic element.
+When a lane is selected, the associated semantic element is selected in the explorer and the semantic element properties are visible in the details view.
+
+
+`CardDescription` allows to define which card will be associated to a lane.
+Like `LaneDescription`, `CardDescription` owns a semantic candidates expression that returns a list of semantic elements. Each semantic element produces a Card displayed in the Lane.
+
+`LaneDescription` and `CardDescription` own *tool descriptions* to define the possible behaviors.
+
+* enable addition
+* enable edition
+* enable hide/show
+* enable deletion
+* drag and drop
+
+==== Understanding the way the representation is rendered
+
+Some tools will do modification in the representation itself such as Hide/Show Lane/Card, Expand/Collapse a Lane.
+
+For the other tools, the change is never done directly on the representation iteself. Instead, the change is done on the semantic model through the operations associated to the tool defined in the view model and then the representation will update based on the current state of the semantic model. 
+For example, when the user drag&drop a card from a lane to another, the expression should update the semantic model in a manner that at the end of the rendering the card get displayed in the lane where it has been dropped.
```

Comment:
the rendering the card get**s** displayed

Reply:
done